README.md

```diff
@@ -1 +1,97 @@
-# Hello-world
\ No newline at end of file
+# HTML & CSS3 Course
```

Comment:
Kindly use the Microverse README template for the README file. and update it with your information. you can find it in this [link](https://github.com/microverseinc/readme-template)

Reply:
@gemmen29 I have made the changes and updated my repository to meet the professional standards. I would kindly request support on the submission of the changes made. Thank you.